package.json

```
@@ -123,7 +117,7 @@
     "eslint": "^8.0.1",
     "eslint-plugin-node": "^11.1.0",
     "factor-bundle": "^2.5.0",
-    "gh-pages": "git+https://github.com/Planeshifter/gh-pages#main",
+    "gh-pages": "git+https://github.com/Planeshifter/gh-pages.git#main",
```

```suggestion
    "gh-pages": "git+https://github.com/Planeshifter/gh-pages#main",
```

@Planeshifter Can you confirm whether the `*.git` file extension is necessary here?